<!DOCTYPE html SYSTEM "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
     <head>
       <title>VarioML</title>
       <style type="text/css">table { border:2px; border-collapse:collapse; }</style>
     <link rel="stylesheet" href="images/xmlns-style.css" type="text/css">
     <link rel="stylesheet" href="images/style.css" type="text/css">
     <link rel="stylesheet" href="images/style2.css" type="text/css">
     <script src="images/toc.js" type="text/javascript"></script>
     <script src="images/smooth.pack.js" type="text/javascript"></script>
     </head>
    <body  onload="generateTOC(document.getElementById('toc'));"><a name="top"></a>
    <table style="border:0; background-color:#ffffff;"><tr><td>
    <IMG src="images/g2p_varioml_title.png"></td>
    <td><H1>VarioML Schema Reference</H1></td></tr></table>
    For more information, see the <a href="https://github.com/VarioML/VarioML">GitHub repository</a>, <a href="http://www.gen2phen.org/groups/varioml">User wiki</a>, and <a href="http://varioml.org">Website</a>.
    <br/><div id="toc"><H3>Contents</H3></div><br/>
    <hr/>
     <h2 class='header'>acknowledgement (<a name='VmlAcknowledgement'><em>VmlAcknowledgement</em></a>)</h2>
     </p><div class='doc'><p>Acknowledgement.</p>
  </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>name</td><td><em>string</em></td><td>mandatory</td><td></td></tr>
         <tr><td>grant_number</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>age (<a name='VmlAge'><em>VmlAge</em></a>)</h2>
</p><div class='text'><h3>text node of type float </h3></div>
     </p><div class='doc'><p>Age at the time of observation.</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>coded</td><td>boolean</td><td>optional</td><td></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>comment (<a name='VmlComment'><em>VmlComment</em></a>)</h2>
     </p><div class='doc'><p>Comments are of type Evidence_ontology. The term 'attribute' is optional. See under description of &quot;Evidence_ontology_term data type.</p>
<p>Comments can contain other comments.</p>
<p><strong>Examples:</strong></p>
<pre><code>    &lt;comment&gt;
        &lt;text&gt;simple comment&lt;/text&gt;
    &lt;/comment&gt;

    &lt;comment source=&quot;example_ontology&quot; term=&quot;example term&quot;&gt;
        &lt;text&gt;my defined comment&lt;/text&gt;
    &lt;/comment&gt;

    &lt;comment&gt;
        &lt;text&gt;whatever&lt;/text&gt;
        &lt;comment&gt;
                &lt;text&gt;my structured comment&lt;/text&gt;
        &lt;/comment&gt;
    &lt;/comment&gt;

    &lt;comment&gt;
        &lt;text&gt;whatever&lt;/text&gt;
        &lt;comment&gt;
            &lt;text&gt;my complex structured comment&lt;/text&gt;
            &lt;evidence_code term=&quot;something&quot;/&gt;
            &lt;db_xref accession=&quot;xyz&quot;/&gt;
        &lt;/comment&gt;
    &lt;/comment&gt; 

    &lt;phenotype term='Autoimmune polyglandular syndrome type 1'&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Hypoparathyroidism&lt;/text&gt;&lt;/comment&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Addison's disease&lt;/text&gt;&lt;/comment&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Mucocutaneous candidiasis&lt;/text&gt;&lt;/comment&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Malabsorption&lt;/text&gt;&lt;/comment&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Chronic active hepatitis&lt;/text&gt;&lt;/comment&gt;
    &lt;/phenotype&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>text</td><td><a href='#VmlText'>VmlText</a></td><td>zero or many</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>consent (<a name='VmlConsent'><em>VmlConsent</em></a>)</h2>
     </p><div class='doc'><p>NOT USED</p>
<p>Database cross-reference for a consent information (not used currently)</p>
  </div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>name</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>consequence (<a name='VmlConsequence'><em>VmlConsequence</em></a>)</h2>
     </p><div class='doc'><p>Consequences of mutation on sequence, structural and molecular levels.</p>
<ul>
<li><strong>Complex frameshift:</strong> Frameshift involving insertions and deletions</li>
<li><strong>Exon deletion:</strong> Deletion encompassing a whole exon or exons, frameshift status unknown</li>
<li><strong>Exon duplication:</strong> Duplication of one of more exons, frameshift status unknown</li>
<li><strong>Frameshift:</strong> Deletion or insertion causing reading frame shift (SO:0000865)</li>
<li>**In-frame: ** (SO:0001587)</li>
<li><strong>In-frame deletion:</strong> Deletion of a whole codon or codons. Can include deletion of one or more exons</li>
<li><strong>In-frame duplication:</strong> A duplication that does not change the reading frame. Can include one or more exons</li>
<li><strong>In-frame insertion:</strong> An insertion of a whole codon or codons. Can include one or more exons</li>
<li><strong>Intronic variant:</strong> A variant in an intron which has not been shown to affect splicing</li>
<li><strong>Missense:</strong> Substitution resulting in a change to a different amino acid (SO:0001783)</li>
<li><strong>Nonsense:</strong> Substitution resulting in a change to a stop codon (SO:0001587)</li>
<li><strong>Out of frame deletion:</strong> Deletion of part of a codon or number of codons resulting in a frameshift. Can include one or more exons</li>
<li><strong>Out of frame duplication:</strong> A duplication that changes the reading frame. Can include one or more exons</li>
<li><strong>Out of frame insertion:</strong> An insertion of part of a codon or number of codons resulting in a frameshift. Can include one or more exons</li>
<li><strong>Silent:</strong> A nucleotide change that does not change the amino acid</li>
<li><strong>Splice site variant:</strong> A mutation that affects splicing</li>
<li><strong>Synonymous:</strong> (SO:0001588)</li>
</ul>
<p><em>Note: implementation of fixed vocabulary for this element is still pending.</em> Sequence ontology and VariO terms are preferred. See: <a href="http://www.gen2phen.org/wiki/lsdb-controlled-vocabulary-terms%22">lsdb-controlled-vocabulary-terms</a> also ClinVarDataDictionary (http://www.ncbi.nlm.nih.gov/projects/clinvar/ClinVarDataDictionary.pdf)</p>
<p>See also documentation under the seq_region and variant_type element</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;consequence source=&quot;SO&quot; accession=&quot;1000071&quot; term=&quot;mutation affecting splicing&quot; /&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>seq_changes (<a name='VmlConsequentVariants'><em>VmlConsequentVariants</em></a>)</h2>
     </p><div class='doc'><p>Related sequence changes on overlapping sequence levels and features.</p>
<p>Element has recursive list of sequence variants of type RNA or AA. The Sub-variant element differ from the main variant element in a following way:</p>
<ul>
<li>type has only values RNA and AA<br /></li>
<li>ref_seq is optional</li>
<li>gene is missing</li>
</ul>
</div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>variant</td><td><a href='#VmlConsequentVariant'>VmlConsequentVariant</a></td><td>one or many</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>contact (<a name='VmlContact'><em>VmlContact</em></a>)</h2>
     </p><div class='doc'></div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>role</td><td>string</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>name</td><td><em>string</em></td><td>mandatory</td><td></td></tr>
         <tr><td>url</td><td><em>string</em></td><td>zero or many</td><td></td></tr>
         <tr><td>address</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>phone</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>fax</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>email</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>creation_date (<a name='VmlCreationDate'><em>VmlCreationDate</em></a>)</h2>
     </p><div class='doc'><p>Date when submitted information was created</p>
</div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td></td><td><em>choice</em></td><td>optional</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>cultivar (<a name='VmlCultivar'><em>VmlCultivar</em></a>)</h2>
     </p><div class='doc'><p>Cultivar</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>db_xref (<a name='VmlDbXRef'><em>VmlDbXRef</em></a>)</h2>
     </p><div class='doc'><p>Abstract data type for database cross references.</p>
<p>Can include other database cross-references and annotations.</p>
<p><strong>Examples:</strong></p>
<pre><code>  &lt;db_xref source=&quot;pubmed&quot; accession=&quot;16965330&quot;/&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>name</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>description (<a name='VmlDescription'><em>VmlDescription</em></a>)</h2>
     </p><div class='doc'></div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>dob (<a name='VmlDobDate'><em>VmlDobDate</em></a>)</h2>
     </p><div class='doc'><p>Date of birth</p>
</div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td></td><td><em>choice</em></td><td>optional</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>embargo_end_date (<a name='VmlEmbargo_end_date'><em>VmlEmbargo_end_date</em></a>)</h2>
</p><div class='text'><h3>text node of type date </h3></div>
     </p><div class='doc'><p>Embargo end date</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>is_undefined</td><td>string</td><td>optional</td><td>true</td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>evidence_code (<a name='VmlEvidenceCode'><em>VmlEvidenceCode</em></a>)</h2>
     </p><div class='doc'><p>Evidence code is an ontology term (i.e. has optional source and accession attributes), which provides information related to the validity or the quality of observation.</p>
<p>dbSNP validation should be used for mutations:</p>
<ul>
<li>Not validated (accession=0)</li>
<li>Multiple reporting (1)</li>
<li>With frequency (2)</li>
<li>Both frequency (3)</li>
<li>Submitter validation (4)</li>
</ul>
<p><strong>Examples:</strong></p>
<pre><code> &lt;evidence_code term=&quot;inferred&quot;/&gt; 

 &lt;evidence_code term=&quot;confirmed&quot; source=&quot;obo.vario&quot;/&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>score</td><td><a href='#VmlScore'>VmlScore</a></td><td>zero or many</td><td></td></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>exon (<a name='VmlExon'><em>VmlExon</em></a>)</h2>
</p><div class='text'><h3>text node of type string </h3></div>
     </p><div class='doc'><p>Exon has optional attributes <strong>source</strong>, <strong>accession</strong>, and <strong>transcript_ref</strong>, which can be used for crossreference purposes. Exon number is given in text node. Element is for legacy exons names</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;exon&gt;01&lt;/exon&gt;   
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>transcript_ref</td><td>string</td><td>optional</td><td></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>factor (<a name='VmlFactor'><em>VmlFactor</em></a>)</h2>
     </p><div class='doc'><p>Possible factors for pathogenicity (under discussion)</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>frequency (<a name='VmlFrequency'><em>VmlFrequency</em></a>)</h2>
     </p><div class='doc'><p>Actual frequency is stored in a subelement depending on value type. This type can be: decimal number, number of cases, or categorized value.</p>
<p>Frequency has the optional attribute 'samples', which can hold information on sample size. Typically this is the number of chromosomes.</p>
<p>Frequency type can be either &quot;allele&quot; or &quot;carrier&quot;, with optional subelements:</p>
<ul>
<li>population</li>
<li>Evidence_ontology_term-type</li>
<li>evidence code</li>
<li>protocol id</li>
<li>comment</li>
</ul>
<p><strong>Examples:</strong></p>
<pre><code> &lt;frequency samples=&quot;1000&quot; &gt;
    &lt;population type=&quot;nationality&quot; term=&quot;Finnish&quot;/&gt;
    &lt;freq&gt;0.2&lt;/freq&gt;
 &lt;/frequency&gt; 

 &lt;frequency&gt;&lt;freq&gt;0.1&lt;/freq&gt;&lt;/frequency&gt;

 &lt;frequency samples=&quot;1000&quot;&gt;
    &lt;cases&gt;11&lt;/cases&gt;
 &lt;/frequency&gt;

 &lt;frequency&gt;
    &lt;population type=&quot;region&quot; term=&quot;Sweden&quot;/&gt;
    &lt;category term=&quot;exist&quot;/&gt;
 &lt;/frequency&gt;      

 &lt;frequency samples=&quot;1111&quot;&gt;
    &lt;population type=&quot;ethnic&quot; term=&quot;Caucasian&quot;/&gt;
    &lt;freq&gt;0.12&lt;/freq&gt;
    &lt;evidence_code source=&quot;sampling_ontology&quot; accession=&quot;abcd&quot; term=&quot;random&quot;/&gt;
 &lt;/frequency&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>samples</td><td>int</td><td>optional</td><td></td></tr>
         <tr><td>type</td><td>string</td><td>optional</td><td>allele | carrier | genotype</td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>population</td><td><a href='#VmlPopulation'>VmlPopulation</a></td><td>optional</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>optional</td><td>freq|counts|category</td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>gender (<a name='VmlGender'><em>VmlGender</em></a>)</h2>
     </p><div class='doc'><p>Gender of individual. Gender code is stored in the attribute <strong>'code'</strong>. Possible values are iso5218 codes: 0,1,2 and 9 ('unknown','male','female' and 'not applicable').</p>
<p>Description of gender (code: 9) can be stored into optional <strong>'description'</strong> sub-element, which is of type #type_LSDB.Evidence_ontology_term&quot; Evidence_ontology_term.</p>
<p><strong>Example:</strong></p>
<pre><code>&lt;gender code='9'&gt;
    &lt;description term=&quot;XX-male&quot;/&gt;
&lt;/gender&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>code</td><td>string</td><td>mandatory</td><td>0 | 1 | 2 | 9</td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><a href='#VmlDescription'>VmlDescription</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>gene (<a name='VmlGene'><em>VmlGene</em></a>)</h2>
     </p><div class='doc'><p>Database cross reference where <strong>'source'</strong> is the name of the database or system (e.g. HUGO) and <strong>'accession'</strong> is the gene name (e.g.AGA).</p>
<p>HGNC names (symbols) should be used to name genes.</p>
<p>Can include other database cross-references and annotations.</p>
<p>Example source: * HGNC</p>
<p>Example accession: * BRCA2</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;gene source=&quot;HGNC&quot; accession=&quot;BRCA2&quot; /&gt;

 &lt;gene source=&quot;HGNC&quot; accession=&quot;BRCA2&quot;&gt;
    &lt;db_xref accession=&quot;7581463&quot; uri=&quot;http://identifiers.org/pubmed/600185&quot; source=&quot;pubmed&quot;/&gt;
    &lt;db_xref accession=&quot;BRCA2&quot; source=&quot;hgnc.symbol&quot; uri=&quot;http://www.identifiers.org/hgnc.symbol/BRCA2&quot;/&gt;
 &lt;/gene&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>name</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>observation (<a name='VmlGenericObservation'><em>VmlGenericObservation</em></a>)</h2>
     </p><div class='doc'><p>Abstract type. See subtypes for the documentation. Observation which has evidence code and observation protocol (dbxref).</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;genetic_origin term=&quot;paternal&quot;&gt;
   &lt;evidence_code term=&quot;inferred&quot;/&gt;            
 &lt;/genetic_origin&gt;

 &lt;phenotype term=&quot;Osteogenesis Imperfecta&quot; &gt;
   &lt;evidence_code term=&quot;assessed by clinician&quot;/&gt;
   &lt;protocol_id source=&quot;protocoldb_xyz&quot; accession=&quot;XYZ0001&quot;/&gt;
 &lt;/phenotype&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
         <tr><td>type</td><td>string</td><td>optional</td><td>lifestyle | environment | relationship</td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>source (<a name='VmlGeneticSource'><em>VmlGeneticSource</em></a>)</h2>
     </p><div class='doc'><p>Source of variant (paternal, maternal). See genetic_origin for more information.</p>
<p>Controlled vocabulary terms for genetic source:</p>
<ul>
<li>paternal</li>
<li>maternal</li>
</ul>
<p><em>N.B: Standardization of this list is pending.</em> See: <a href="http://www.gen2phen.org/wiki/lsdb-controlled-vocabulary-terms">lsdb-controlled-vocabulary-terms</a></p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>population</td><td><a href='#VmlPopulation'>VmlPopulation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>genetic_origin (<a name='VmlGenetic_origin'><em>VmlGenetic_origin</em></a>)</h2>
     </p><div class='doc'><p>Genetic origin of variation.</p>
<p>Genetic origin corresponds to the <a href="http://www.sequenceontology.org/wiki/index.php/Category:SO:0001762_!_variant_origin">Sequence Ontology's variant_origin</a>.</p>
<p>Genetic origin and source are evidence ontology terms.</p>
<p>Genetic origin documents the following information:</p>
<ul>
<li>term - ontology term for origin of variant (denovo, sporadic, inherited)</li>
<li>source - parental origin of variation (paternal, maternal)</li>
<li>evidence code - evidence code for the observation</li>
<li>list of individuals</li>
</ul>
<p><strong>Controlled vocabulary terms for genetic origin:</strong></p>
<ul>
<li>inherited</li>
<li>sporadic</li>
<li>de novo</li>
</ul>
<p><em>N.B: Standardization of this list is pending.</em> See: <a href="http://www.gen2phen.org/wiki/lsdb-controlled-vocabulary-terms">lsdb-controlled-vocabulary-terms</a></p>
<p><strong>Example:</strong></p>
<pre><code> &lt;genetic_origin term=&quot;inherited&quot;&gt;
    &lt;source term=&quot;paternal&quot;/&gt;
    &lt;evidence_code term=&quot;inferred&quot;/&gt;                                
 &lt;/genetic_origin&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>source</td><td><a href='#VmlGeneticSource'>VmlGeneticSource</a></td><td>optional</td><td></td></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>genotype (<a name='VmlGenotype'><em>VmlGenotype</em></a>)</h2>
     </p><div class='doc'><p>Genotype</p>
<p>Orginal genotype call e.g. A/T</p>
</div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>call</td><td><em>string</em></td><td>mandatory</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>group_type (<a name='VmlGroupType'><em>VmlGroupType</em></a>)</h2>
     </p><div class='doc'><p>Group type (NOT USED)</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>haplotype (<a name='VmlHaplotype'><em>VmlHaplotype</em></a>)</h2>
     </p><div class='doc'><p>Haplotypes. Complex variants can be given as a list of phased variants. This element still in testing.</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>id</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>allele</td><td>int</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>name</td><td><a href='#VmlName'>VmlName</a></td><td>optional</td><td></td></tr>
         <tr><td>variant</td><td><a href='#VmlVariantEvent'>VmlVariantEvent</a></td><td>one or many</td><td></td></tr>
         <tr><td>sequence</td><td><a href='#VmlSequence'>VmlSequence</a></td><td>optional</td><td></td></tr>
         <tr><td>consequence</td><td><a href='#VmlConsequence'>VmlConsequence</a></td><td>zero or many</td><td></td></tr>
         <tr><td>pathogenicity</td><td><a href='#VmlPathogenicity'>VmlPathogenicity</a></td><td>zero or many</td><td></td></tr>
         <tr><td>frequency</td><td><a href='#VmlFrequency'>VmlFrequency</a></td><td>zero or many</td><td></td></tr>
         <tr><td>seq_changes</td><td><a href='#VmlConsequentVariants'>VmlConsequentVariants</a></td><td>optional</td><td></td></tr>
         <tr><td>aliases</td><td><a href='#VmlRelatedVariants'>VmlRelatedVariants</a></td><td>optional</td><td></td></tr>
         <tr><td>source</td><td><a href='#VmlSource'>VmlSource</a></td><td>optional</td><td></td></tr>
         <tr><td>location</td><td><a href='#VmlLocation'>VmlLocation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>individual (<a name='VmlIndividual'><em>VmlIndividual</em></a>)</h2>
     </p><div class='doc'><p>Individual or patient</p>
<p><strong>Example:</strong></p>
<pre><code>    &lt;individual&gt;
         &lt;gender code=&quot;2&quot; /&gt;
         &lt;phenotype term=&quot;Osteogenesis Imperfecta&quot; &gt;
         &lt;/phenotype&gt;
         &lt;variant type=&quot;cDNA&quot;&gt;
             &lt;gene source=&quot;HGNC&quot; accession=&quot;COL1A1&quot; /&gt;
             &lt;ref_seq accession=&quot;NG_007400.1&quot; /&gt;
             &lt;name scheme=&quot;HGVS&quot;&gt;c.579delT&lt;/name&gt;
             &lt;genetic_origin term=&quot;paternal&quot;&gt;
                 &lt;evidence_code term=&quot;inferred&quot;/&gt;            
             &lt;/genetic_origin&gt;
             &lt;sharing_policy type=&quot;openAccess&quot;&gt;
                 &lt;embargo_end_date&gt;2002-12-12&lt;/embargo_end_date&gt;
                 &lt;use_permission accession=&quot;CC0&quot; uri=&quot;http://creativecommons.org/publicdomain/zero/1.0/&quot; term=&quot;Creative commons&quot;/&gt;
             &lt;/sharing_policy&gt;

             &lt;comment&gt;
                 &lt;text&gt;Variant inherited from affected father&lt;/text&gt;
             &lt;/comment&gt;

         &lt;/variant&gt;
     &lt;/individual&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>id</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>gender</td><td><a href='#VmlGender'>VmlGender</a></td><td>optional</td><td></td></tr>
         <tr><td>dob</td><td><a href='#VmlDobDate'>VmlDobDate</a></td><td>optional</td><td></td></tr>
         <tr><td>age</td><td><a href='#VmlAge'>VmlAge</a></td><td>optional</td><td></td></tr>
         <tr><td>original_id</td><td><a href='#VmlOriginalId'>VmlOriginalId</a></td><td>optional</td><td></td></tr>
         <tr><td>role</td><td><a href='#VmlRoleType'>VmlRoleType</a></td><td>optional</td><td></td></tr>
         <tr><td>relationship</td><td><a href='#VmlRelationship'>VmlRelationship</a></td><td>zero or many</td><td></td></tr>
         <tr><td>organism</td><td><a href='#VmlOrganism'>VmlOrganism</a></td><td>optional</td><td></td></tr>
         <tr><td>strain</td><td><a href='#VmlStrain'>VmlStrain</a></td><td>optional</td><td></td></tr>
         <tr><td>cultivar</td><td><a href='#VmlCultivar'>VmlCultivar</a></td><td>optional</td><td></td></tr>
         <tr><td>phenotype</td><td><a href='#VmlPhenotype'>VmlPhenotype</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation</td><td><a href='#VmlGenericObservation'>VmlGenericObservation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>population</td><td><a href='#VmlPopulation'>VmlPopulation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant</td><td><a href='#VmlVariant'>VmlVariant</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant_group</td><td><a href='#VmlVariantGroup'>VmlVariantGroup</a></td><td>zero or many</td><td></td></tr>
         <tr><td>source</td><td><a href='#VmlSource'>VmlSource</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td>sharing_policy</td><td><a href='#VmlSharingPolicy'>VmlSharingPolicy</a></td><td>optional</td><td></td></tr>
         <tr><td>creation_date</td><td><a href='#VmlCreationDate'>VmlCreationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>modification_date</td><td><a href='#VmlModificationDate'>VmlModificationDate</a></td><td>optional</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>inheritance_pattern (<a name='VmlInheritancePattern'><em>VmlInheritancePattern</em></a>)</h2>
     </p><div class='doc'><p>Inheritance pattern of the phenotype. Possible values for the term:</p>
<ul>
<li>familial<br /></li>
<li>familial, consanguineous parents<br /></li>
<li>familial, autosomal dominant</li>
<li>familial, autosomal recessive<br /></li>
<li>familial, Xlinked<br /></li>
<li>sporadic<br /></li>
<li>sporadic, consanguineous parents<br /></li>
<li>sporadic, consanguineous parents (1st degree)<br /></li>
<li>sporadic, consanguineous parents (2nd degree)<br /></li>
<li>sporadic, consanguineous parents (3rd degree)<br /></li>
<li>sporadic, nonconsanguineous parents<br /></li>
<li>sporadic, consanguinity parents?<br /></li>
<li>sporadic? (parents not tested)</li>
</ul>
<p>See: <a href="http://www.gen2phen.org/wiki/lsdb-controlled-vocabulary-terms">lsdb-controlled-vocabulary-terms</a></p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>location (<a name='VmlLocation'><em>VmlLocation</em></a>)</h2>
     </p><div class='doc'><p>Location of variant on a reference sequence/genomic build.</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;location&gt;
   &lt;ref_seq source=&quot;embl&quot; accession=&quot;ABC1023345&quot;/&gt;
   &lt;start&gt;111111&lt;/start&gt;
   &lt;end&gt;111112&lt;/end&gt;
 &lt;/location&gt;



 &lt;location&gt;
    &lt;ref_seq source=&quot;refseq&quot; accession=&quot;NC_000001.10&quot;/&gt;
    &lt;start&gt;100315752&lt;/ns2:start&gt;
    &lt;end&gt;100315752&lt;/ns2:end&gt;
 &lt;/location&gt;
</code></pre>
 
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>unit</td><td>string</td><td>optional</td><td>feature | residue</td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>ref_seq</td><td><a href='#VmlRefSeq'>VmlRefSeq</a></td><td>optional</td><td></td></tr>
         <tr><td>chr</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>start</td><td><em>string</em></td><td>mandatory</td><td></td></tr>
         <tr><td>end</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>lsdb (<a name='VmlLsdb'><em>VmlLsdb</em></a>)</h2>
     </p><div class='doc'><p>Single data submission batch element for individual data. The batch contains:</p>
<ul>
<li>id,uri - optional submission attributes<br /></li>
<li>created - optional timestamp when data submission batch was created<br /></li>
<li>source - information on source(s) of submission. Submitting source</li>
<li>list of individuals</li>
</ul>
<p><strong>Example:</strong></p>
<pre><code> &lt;lsdb xmlns=&quot;http://gen2phen.org/varioml/2.0&quot; xmlns:xsi=&quot;http://www.w3.org/2001/XMLSchema-instance&quot;
     xmlns:dc=&quot;example.org&quot; xsi:noNamespaceSchemaLocation=&quot;lsdb.xsd&quot;&gt;
     &lt;created&gt;2000-01-12T12:13:14Z&lt;/created&gt;
     &lt;source id=&quot;AIREbase&quot; version=&quot;1.2&quot; uri=&quot;http://www.uta.fi/imt/bioinfo/AIREbase/&quot;&gt;
         &lt;name&gt;DatabaseXYZ&lt;/name&gt;
         &lt;url&gt;http://www.uta.fi/imt/bioinfo/AIREbase/&lt;/url&gt;
         &lt;contact&gt;
             &lt;name&gt;Matti Meikäläinen&lt;/name&gt;
             &lt;address&gt;Institute for Molecular Medicine&lt;/address&gt;
         &lt;/contact&gt;
         &lt;acknowledgement&gt;
             &lt;name&gt;European Union&lt;/name&gt;
             &lt;grant_number accession=&quot;23230001&quot;/&gt;
         &lt;/acknowledgement&gt;
     &lt;/source&gt;

     &lt;individual id=&quot;xyz&quot;&gt;
         &lt;gender code=&quot;2&quot;/&gt;
         &lt;variant&gt;
             &lt;ref_seq accession=&quot;xyz&quot;&gt;&lt;/ref_seq&gt;
             &lt;name scheme=&quot;HGVS&quot;&gt;c.1343GT&lt;/name&gt;
         &lt;/variant&gt;
     &lt;/individual&gt;

     &lt;individual id=&quot;abce&quot;&gt;
         &lt;gender code=&quot;2&quot;/&gt;
         &lt;variant&gt;
             &lt;ref_seq accession=&quot;xyz&quot;&gt;&lt;/ref_seq&gt;
             &lt;name scheme=&quot;HGVS&quot;&gt;c.1343GC&lt;/name&gt;
         &lt;/variant&gt;
     &lt;/individual&gt;

 &lt;/lsdb&gt;
</code></pre>
 </div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>id</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>schema_version</td><td>decimal</td><td>optional</td><td></td></tr>
         <tr><td>submissionid_type</td><td>string</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>created</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>source</td><td><a href='#VmlSource'>VmlSource</a></td><td>zero or many</td><td></td></tr>
         <tr><td>individual</td><td><a href='#VmlIndividual'>VmlIndividual</a></td><td>zero or many</td><td></td></tr>
         <tr><td>panel</td><td><a href='#VmlPanel'>VmlPanel</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant</td><td><a href='#VmlVariant'>VmlVariant</a></td><td>zero or many</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>modification_date (<a name='VmlModificationDate'><em>VmlModificationDate</em></a>)</h2>
     </p><div class='doc'><p>Date when submitted information was modified.</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;modification_date&gt;2001&lt;/modification_date&gt;
 &lt;modification_date&gt;2001-09&lt;/modification_date&gt;
 &lt;modification_date&gt;2001-09-10&lt;/modification_date&gt;
</code></pre>
</div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td></td><td><em>choice</em></td><td>optional</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>name (<a name='VmlName'><em>VmlName</em></a>)</h2>
</p><div class='text'><h3>text node of type string </h3></div>
     </p><div class='doc'><p>Name of variant</p>
<p>Element has following optional attributes: <strong>scheme</strong> - naming scheme, which should typically be HGVS.</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;name type=&quot;DNA&quot; scheme=&quot;HGVS&quot;&gt;
      c.755G\A
 &lt;/name&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>scheme</td><td>string</td><td>optional</td><td></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>observations (<a name='VmlObservationCombo'><em>VmlObservationCombo</em></a>)</h2>
     </p><div class='doc'><p>Abstract type. See subtypes for the documentation. Observation which has evidence code and observation protocol (dbxref).</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;genetic_origin term=&quot;paternal&quot;&gt;
   &lt;evidence_code term=&quot;inferred&quot;/&gt;            
 &lt;/genetic_origin&gt;

 &lt;phenotype term=&quot;Osteogenesis Imperfecta&quot; &gt;
   &lt;evidence_code term=&quot;assessed by clinician&quot;/&gt;
   &lt;protocol_id source=&quot;protocoldb_xyz&quot; accession=&quot;XYZ0001&quot;/&gt;
 &lt;/phenotype&gt;
</code></pre>
</div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td></td><td><em>choice</em></td><td>optional</td><td>panel|individual</td></tr>
         <tr><td>phenotype</td><td><a href='#VmlPhenotype'>VmlPhenotype</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant</td><td><a href='#VmlVariant'>VmlVariant</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation</td><td><a href='#VmlGenericObservation'>VmlGenericObservation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant_group</td><td><a href='#VmlVariantGroup'>VmlVariantGroup</a></td><td>zero or many</td><td></td></tr>
         <tr><td>sharing_policy</td><td><a href='#VmlSharingPolicy'>VmlSharingPolicy</a></td><td>optional</td><td></td></tr>
         <tr><td>creation_date</td><td><a href='#VmlCreationDate'>VmlCreationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>modification_date</td><td><a href='#VmlModificationDate'>VmlModificationDate</a></td><td>optional</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>observation_date (<a name='VmlObservationDate'><em>VmlObservationDate</em></a>)</h2>
     </p><div class='doc'><p>Observation_date captures the date of an observation. A boolean attribute can be used when an exact date cannot be given. An age attribute captures the age of the subject at the time of observation.</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>coded</td><td>boolean</td><td>optional</td><td></td></tr>
         <tr><td>date</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>age</td><td>float</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>category (<a name='VmlOntologyTerm'><em>VmlOntologyTerm</em></a>)</h2>
     </p><div class='doc'></div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>organism (<a name='VmlOrganism'><em>VmlOrganism</em></a>)</h2>
     </p><div class='doc'><p>Organism</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>original_id (<a name='VmlOriginalId'><em>VmlOriginalId</em></a>)</h2>
     </p><div class='doc'><p>Identifier used by the submitter.</p>
  </div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>name</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>panel (<a name='VmlPanel'><em>VmlPanel</em></a>)</h2>
     </p><div class='doc'><p>Panel. Unspecified group of individuals. Has following information besides properties inherited from AbstractObservation target:</p>
<p>*individuals - list of individuals beloning to the panel</p>
<p><strong>Example:</strong></p>
<p>Frequency for existence of two alleles.</p>
<pre><code>    &lt;panel&gt;
         &lt;frequency samples=&quot;224&quot; type=&quot;allele&quot;&gt;
             &lt;freq&gt;0.763&lt;/freq&gt;
         &lt;/frequency&gt;
         &lt;population type=&quot;group&quot; term=&quot;El Salvadorans&quot;/&gt;
         &lt;population type=&quot;ethnic&quot; term=&quot;Mestizo&quot;&gt;
             &lt;comment&gt;&lt;text&gt;Amerindian and European descent&lt;/text&gt;&lt;/comment&gt;
         &lt;/population&gt;
         &lt;variant&gt;
             &lt;ref_seq accession=&quot;NM_000777&quot;/&gt;
             &lt;name&gt;g.6986AG&lt;/name&gt;                                    
         &lt;/variant&gt;
         &lt;variant&gt;
             &lt;ref_seq accession=&quot;NM_000777&quot;/&gt;
             &lt;name&gt;g.1289TG&lt;/name&gt;                                    
         &lt;/variant&gt;
     &lt;/panel&gt;
</code></pre>
<p>Note also that also variant specific frequency can be given as usual:</p>
<pre><code>     &lt;panel&gt;      
         &lt;population type=&quot;group&quot; term=&quot;Yorubans&quot;/&gt;
         &lt;population type=&quot;region&quot; term=&quot;Nigeria&quot;/&gt;
         &lt;variant&gt;
             &lt;ref_seq accession=&quot;NM_000777&quot;/&gt;
             &lt;name&gt;g.6986AG&lt;/name&gt;                                                
             &lt;frequency samples=&quot;50&quot; type=&quot;allele&quot;&gt; 
                 &lt;freq&gt;0.06&lt;/freq&gt;
             &lt;/frequency&gt;
         &lt;/variant&gt;
     &lt;/panel&gt;
</code></pre>
<p><strong>Usage of the format must be agreed on separately.</strong></p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>size</td><td>integer</td><td>optional</td><td></td></tr>
         <tr><td>type</td><td>string</td><td>optional</td><td>family | extended family | population</td></tr>
         <tr><td>id</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>individual</td><td><a href='#VmlIndividual'>VmlIndividual</a></td><td>zero or many</td><td></td></tr>
         <tr><td>gender</td><td><a href='#VmlGender'>VmlGender</a></td><td>optional</td><td></td></tr>
         <tr><td>dob</td><td><a href='#VmlDobDate'>VmlDobDate</a></td><td>optional</td><td></td></tr>
         <tr><td>age</td><td><a href='#VmlAge'>VmlAge</a></td><td>optional</td><td></td></tr>
         <tr><td>original_id</td><td><a href='#VmlOriginalId'>VmlOriginalId</a></td><td>optional</td><td></td></tr>
         <tr><td>role</td><td><a href='#VmlRoleType'>VmlRoleType</a></td><td>optional</td><td></td></tr>
         <tr><td>relationship</td><td><a href='#VmlRelationship'>VmlRelationship</a></td><td>zero or many</td><td></td></tr>
         <tr><td>organism</td><td><a href='#VmlOrganism'>VmlOrganism</a></td><td>optional</td><td></td></tr>
         <tr><td>strain</td><td><a href='#VmlStrain'>VmlStrain</a></td><td>optional</td><td></td></tr>
         <tr><td>cultivar</td><td><a href='#VmlCultivar'>VmlCultivar</a></td><td>optional</td><td></td></tr>
         <tr><td>phenotype</td><td><a href='#VmlPhenotype'>VmlPhenotype</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation</td><td><a href='#VmlGenericObservation'>VmlGenericObservation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>population</td><td><a href='#VmlPopulation'>VmlPopulation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant</td><td><a href='#VmlVariant'>VmlVariant</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant_group</td><td><a href='#VmlVariantGroup'>VmlVariantGroup</a></td><td>zero or many</td><td></td></tr>
         <tr><td>source</td><td><a href='#VmlSource'>VmlSource</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td>sharing_policy</td><td><a href='#VmlSharingPolicy'>VmlSharingPolicy</a></td><td>optional</td><td></td></tr>
         <tr><td>creation_date</td><td><a href='#VmlCreationDate'>VmlCreationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>modification_date</td><td><a href='#VmlModificationDate'>VmlModificationDate</a></td><td>optional</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>pathogenicity (<a name='VmlPathogenicity'><em>VmlPathogenicity</em></a>)</h2>
     </p><div class='doc'><p>Pathogenicity of variant.</p>
<p>Controlled vocabulary for pathogenicity is sourced from the GEN2PHEN project. See: <a href="http://www.gen2phen.org/wiki/varioml-pathogenicity">VarioML - Pathogenicity</a>.</p>
<p>SKOS terms are preferred: http://purl.org/varioml/pathogenicity/skos/1.0/</p>
<pathogenicity term="Likely pathogenic"  uri="http://purl.org/varioml/pathogenicity/skos/1.0#p_0003"> </pathogenicity>
<pathogenicity term="Likely pathogenic"  source="g2p" accession="p_0003" > </pathogenicity>
<p>Note that g2p is always used as a source</p>
<p>Examples of the terms:</p>
<ul>
<li>Non-pathogenic</li>
<li>Not known</li>
<li>Pathogenic</li>
<li>Probably not pathogenic</li>
<li>Probably pathogenic</li>
<li>Unclassified</li>
</ul>
<p><em>Pathogenicity</em> has new attributes: <strong>phenotype</strong>, <strong>scope</strong> <em>(individual, family, population)</em> and <strong>score</strong>.</p>
<p><strong>CMGS/VKGL suggested classification</strong></p>
<p>Via <a href="http://www.interactive-biosoftware.com/alamut/doc/CMGS-VKGL-Classification.html">Alamut</a>:</p>
<p>The <a href="http://cmgsweb.shared.hosting.zen.co.uk/BPGs/pdfs%20current%20bpgs/UV%20GUIDELINES%20ratified.pdf">Practice guidelines for the Interpretation and Reporting of Unclassified Variants (UVs) in Clinical Molecular Genetics</a> suggest the following classification system:</p>
<p>Class 1 – Certainly not pathogenic Class 2 – Unlikely to be pathogenic but cannot be formally proven Class 3 – Likely to be pathogenic but cannot be formally proven Class 4 - Certainly pathogenic</p>
<hr />
<p><em>Note [1]: implementation of fixed vocabulary for this element is still pending.</em> See: <a href="http://www.gen2phen.org/wiki/lsdb-controlled-vocabulary-terms%22">lsdb-controlled-vocabulary-terms</a></p>
<p>*Note [2]: Textual values will likely be changing to &quot;Affects function&quot; to cover non-disease implications such as <em>Eye color</em>.</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>scope</td><td>string</td><td>optional</td><td>individual | family | population</td></tr>
         <tr><td>panel_ref</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>phenotype</td><td><a href='#VmlPhenotype'>VmlPhenotype</a></td><td>zero or many</td><td></td></tr>
         <tr><td>factor</td><td><a href='#VmlFactor'>VmlFactor</a></td><td>zero or many</td><td></td></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>phenotype (<a name='VmlPhenotype'><em>VmlPhenotype</em></a>)</h2>
     </p><div class='doc'><p>Phenotype of individual. Phenotype is evidence ontology term (see evidence_ontology)</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;phenotype term='Autoimmune polyglandular syndrome type 1'/&gt;

 &lt;phenotype term='Autoimmune polyglandular syndrome type 1' accession='240300' source='omim'&gt;
     &lt;evidence_code term=&quot;confirmed&quot; source=&quot;vario&quot;/&gt;
     &lt;protocol_id accession=&quot;observation_method_xyz&quot; source=&quot;protocol_database&quot;/&gt;
    &lt;db_xref accession=&quot;12345&quot; source=&quot;pubmed&quot;/&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Hypoparathyroidism&lt;/text&gt;&lt;/comment&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Addison's disease&lt;/text&gt;&lt;/comment&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Mucocutaneous candidiasis&lt;/text&gt;&lt;/comment&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Malabsorption&lt;/text&gt;&lt;/comment&gt;
     &lt;comment term='symptom'&gt;&lt;text&gt;Chronic active hepatitis&lt;/text&gt;&lt;/comment&gt;
 &lt;/phenotype&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>type</td><td>string</td><td>optional</td><td>symptom | diagnose</td></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>inheritance_pattern</td><td><a href='#VmlInheritancePattern'>VmlInheritancePattern</a></td><td>optional</td><td></td></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>population (<a name='VmlPopulation'><em>VmlPopulation</em></a>)</h2>
     </p><div class='doc'><p>Information related to population group the individual belongs to.</p>
<p>Population element has the &quot;type&quot; attribute which indicates the selection criteria of the population, using the values:</p>
<ul>
<li>ethnic</li>
<li>region</li>
<li>race</li>
<li>other</li>
<li>unknown</li>
</ul>
<p>The attribute is informative: the actual type of population should be defined in the source ontology (if it is used).</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;population type=&quot;ethnic&quot; term=&quot;Caucasian&quot;/&gt;                    

 &lt;population type=&quot;ethnic&quot; term=&quot;Caucasian&quot;&gt; 
    &lt;evidence_code source=&quot;example_ontology_term&quot; accession=&quot;abcd&quot; term=&quot;ethnicity of mother&quot;/&gt;
 &lt;/population&gt;

 &lt;population type=&quot;ethnic&quot; term=&quot;Asian&quot; &gt; 
    &lt;evidence_code source=&quot;example_ontology_term&quot; accession=&quot;abcd&quot; term=&quot;ethnicity of father&quot;/&gt;
 &lt;/population&gt;

 &lt;population type=&quot;region&quot; term=&quot;Ahvenanmaa&quot; source=&quot;ISO 3166&quot;  accession=&quot;ISO 3166-2:AX&quot;&gt; 
    &lt;evidence_code term=&quot;place of birth&quot;/&gt;
 &lt;/population&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>type</td><td>string</td><td>mandatory</td><td>region | nationality | ethnic | race | organism | group | primary_language | language_family | other | unknown</td></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>protocol_id (<a name='VmlProtocolId'><em>VmlProtocolId</em></a>)</h2>
     </p><div class='doc'><p>Cross-reference to a protocol database.</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>name</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>ref_seq (<a name='VmlRefSeq'><em>VmlRefSeq</em></a>)</h2>
     </p><div class='doc'><p>Database cross reference for reference sequence.</p>
<p>Can include other database cross-references and annotations.</p>
<p>Use <a href="http://www.ncbi.nlm.nih.gov/genbank/">Genbank</a>to source reference sequences.</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;ref_seq source=&quot;genbank&quot; accession=&quot;NG_007088.1&quot;/&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>name</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>aliases (<a name='VmlRelatedVariants'><em>VmlRelatedVariants</em></a>)</h2>
     </p><div class='doc'><p>Aliases or alternative legacy definitions (names) of the variant.</p>
<p>Element has list of sequence variants (recursive data structure) which can be seen as aliases of the main variant. Typically the variants are annotated against different reference sequences, and/or their name is based on a different naming scheme.</p>
<p><strong>Example:</strong></p>
<pre><code>   &lt;variant&gt;
        &lt;ref_seq source=&quot;genbank&quot; accession=&quot;NG_001337.2&quot;&gt;&lt;/ref_seq&gt;
        &lt;name scheme=&quot;HGVS&quot;&gt;g.22048CG&lt;/name&gt;
        &lt;aliases&gt;
            &lt;variant&gt;
                &lt;ref_seq accession=&quot;NG_002335.2&quot;/&gt;
                &lt;name&gt;g.1400GC&lt;/name&gt;                    
            &lt;/variant&gt;
            &lt;variant&gt;
                &lt;ref_seq accession=&quot;NG_007072.2&quot;/&gt;
                &lt;name&gt;g.14034GC&lt;/name&gt;
            &lt;/variant&gt;
            &lt;variant&gt;
                &lt;ref_seq accession=&quot;NG_007088.1&quot;/&gt;
                &lt;name&gt;g.28259GC&lt;/name&gt;
            &lt;/variant&gt;                
        &lt;/aliases&gt;

        &lt;db_xref  source=&quot;dbsnp&quot; accession=&quot;rs13&quot;/&gt;
    &lt;/variant&gt;
</code></pre>
</div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>variant</td><td><a href='#VmlVariant'>VmlVariant</a></td><td>one or many</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>relationship (<a name='VmlRelationship'><em>VmlRelationship</em></a>)</h2>
     </p><div class='doc'>relationships .. NOT NEEDED
<p>Relationship</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>individual</td><td><a href='#VmlIndividual'>VmlIndividual</a></td><td>zero or many</td><td></td></tr>
         <tr><td>panel</td><td><a href='#VmlPanel'>VmlPanel</a></td><td>zero or many</td><td></td></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>restriction_site (<a name='VmlRestrictionSite'><em>VmlRestrictionSite</em></a>)</h2>
     </p><div class='doc'><p>Restriction enzyme site.</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>role (<a name='VmlRoleType'><em>VmlRoleType</em></a>)</h2>
     </p><div class='doc'><p>Role in a group</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>sample (<a name='VmlSample'><em>VmlSample</em></a>)</h2>
     </p><div class='doc'><p>Local sample id</p>
  </div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>name</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>score (<a name='VmlScore'><em>VmlScore</em></a>)</h2>
     </p><div class='doc'><p>Score</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>value</td><td><em>string</em></td><td>mandatory</td><td></td></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>seq_region (<a name='VmlSeqRegion'><em>VmlSeqRegion</em></a>)</h2>
     </p><div class='doc'><p>Name of the underlying sequence reqion. Sequence Ontology (SO) terms are used to standardise attributes such as <em>exon, intron, 3' UTR</em>, etc.</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>seq_region</td><td><a href='#VmlSeqRegion'>VmlSeqRegion</a></td><td>zero or many</td><td></td></tr>
         <tr><td>location</td><td><a href='#VmlLocation'>VmlLocation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>sequence (<a name='VmlSequence'><em>VmlSequence</em></a>)</h2>
     </p><div class='doc'><p>Reference and variant sequence strings. Accession number of reference sequence is given in ref_seq element (see the variant)</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;sequence&gt;
     &lt;reference&gt;atttgatcgttc&lt;/reference&gt;
     &lt;variant&gt;atttgGtcgttc&lt;/variant&gt;
 &lt;/sequence&gt;
</code></pre>
</div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>reference</td><td><em>string</em></td><td>mandatory</td><td></td></tr>
         <tr><td>variant</td><td><em>string</em></td><td>mandatory</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>sharing_policy (<a name='VmlSharingPolicy'><em>VmlSharingPolicy</em></a>)</h2>
     </p><div class='doc'><p>Data sharing policy. Following access policy types are used: closedAccess,embargoedAccess,restrictedAccess and openAccess</p>
<p>Adopted from <a href="http://wiki.surffoundation.nl/display/standards/info-eu-repo/#info-eu-repo-AccessRights">OpenAIRE specification</a></p>
<p><strong>Example:</strong></p>
<pre><code>    &lt;sharing_policy type=&quot;openAccess&quot;&gt;
        &lt;embargo_end_date&gt;2002-12-12&lt;/embargo_end_date&gt;
        &lt;use_permission accession=&quot;CC0&quot; uri=&quot;http://creativecommons.org/publicdomain/zero/1.0/&quot; term=&quot;Creative commons&quot;/&gt;
    &lt;/sharing_policy&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>type</td><td>string</td><td>mandatory</td><td>closedAccess | embargoedAccess | restrictedAccess | openAccess</td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>embargo_end_date</td><td><a href='#VmlEmbargo_end_date'>VmlEmbargo_end_date</a></td><td>optional</td><td></td></tr>
         <tr><td>use_permission</td><td><a href='#VmlUsePermission'>VmlUsePermission</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>source (<a name='VmlSource'><em>VmlSource</em></a>)</h2>
     </p><div class='doc'><p>Contains information on data source. Typically specifies the source database, although the system, organization, or institution from which data is exported can also be used.</p>
<p>In the LSDB XML format there can be more than one source, so that information on original data providers can be included.</p>
<p>Element is re-used from the <a href="http://www.lrg-sequence.org">LRG specification</a>.</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;source id=&quot;dmudb&quot;&gt;   
    &lt;name&gt;DMuDB, NGRL, Manchester&lt;/name&gt; 
    &lt;url&gt;http://www.ngrl.org.uk/Manchester/projects/informatics/dmudb&lt;/url&gt;  
    &lt;contact&gt;
        &lt;name&gt;Glen Dobson&lt;/name&gt;
        &lt;address&gt;NGRL, Genetic Medicine, St Mary's Hospital, Oxford Road, Manchester M13 9WL&lt;/address&gt;
        &lt;phone&gt;(+44) 0000 000 0000&lt;/phone&gt;
        &lt;fax&gt;(+44) 0000 000 0000&lt;/fax&gt;
        &lt;email&gt;example@nnn.nn.nnn&lt;/email&gt;
    &lt;/contact&gt;
 &lt;/source&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>id</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>version</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>date</td><td>date</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>name</td><td><em>string</em></td><td>mandatory</td><td></td></tr>
         <tr><td>url</td><td><em>string</em></td><td>zero or many</td><td></td></tr>
         <tr><td>contact</td><td><a href='#VmlContact'>VmlContact</a></td><td>zero or many</td><td></td></tr>
         <tr><td>acknowledgement</td><td><a href='#VmlAcknowledgement'>VmlAcknowledgement</a></td><td>zero or many</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>strain (<a name='VmlStrain'><em>VmlStrain</em></a>)</h2>
     </p><div class='doc'><p>Strain</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>cafe_variome (<a name='VmlSubmission'><em>VmlSubmission</em></a>)</h2>
     </p><div class='doc'></div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>id</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>schema_version</td><td>decimal</td><td>optional</td><td></td></tr>
         <tr><td>submissionid_type</td><td>string</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>created</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>source</td><td><a href='#VmlSource'>VmlSource</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant</td><td><a href='#VmlVariant'>VmlVariant</a></td><td>zero or many</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>submitter_id (<a name='VmlSubmitterId'><em>VmlSubmitterId</em></a>)</h2>
     </p><div class='doc'><p>Randomized identifier of individual as used by submitter.</p>
  </div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>name</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>text (<a name='VmlText'><em>VmlText</em></a>)</h2>
</p><div class='text'><h3>text node of type string </h3></div>
     </p><div class='doc'><p>Text</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>content_type</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>lang</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>encoding</td><td>string</td><td>optional</td><td></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>tissue (<a name='VmlTissue'><em>VmlTissue</em></a>)</h2>
     </p><div class='doc'><p>Tissue</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>tissue_distribution (<a name='VmlTissueDistribution'><em>VmlTissueDistribution</em></a>)</h2>
     </p><div class='doc'><p>Tissue distribution</p>
<p>Controlled vocabulary terms for tissue distribution:</p>
<ul>
<li>constitutional</li>
<li>mosaic</li>
<li>mosaic in germline</li>
</ul>
<p><em>Note: implementation of fixed vocabulary for this element is still pending.</em> See: <a href="http://www.gen2phen.org/wiki/lsdb-controlled-vocabulary-terms%22">lsdb-controlled-vocabulary-terms</a></p>
<p><strong>Example:</strong></p>
<pre><code> &lt;tissue_distribution term=&quot;mosaic&quot;/&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>use_permission (<a name='VmlUsePermission'><em>VmlUsePermission</em></a>)</h2>
     </p><div class='doc'><p>Licensing information.</p>
<p><strong>Example:</strong></p>
<pre><code> &lt;use_permission accession=&quot;CC0&quot; uri=&quot;http://creativecommons.org/publicdomain/zero/1.0/&quot; term=&quot;Creative commons&quot;/&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>value (<a name='VmlValue'><em>VmlValue</em></a>)</h2>
     </p><div class='doc'><p>A name/value pair can be added to an observation using the <em>value</em> element. E.g. the number of exons, residue, etc.</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
         <tr><td>unit</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>type</td><td>string</td><td>optional</td><td>integer | float | text | date | datetime</td></tr>
         <tr><td>val</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>nval</td><td>double</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>variant (<a name='VmlVariant'><em>VmlVariant</em></a>)</h2>
     </p><div class='doc'><p>Variant. Typically has the following information:</p>
<ul>
<li>type - DNA,cDNA,RNA,AA<br /></li>
<li>gene</li>
<li>ref_seq - reference sequence</li>
<li>name - name of variant, preferrably using the HGVS nomenclature<br /></li>
<li>consequence - consequence (list of consequences) of the variant using for example sequence or variation ontology</li>
<li>pathogenicity</li>
<li>phenotype - known phenotypic consequence</li>
<li>variant_detection - variant detection method</li>
<li>restriction_site - site of restriction enzyme</li>
<li>seq_changes - effects of variant on overlapping sequence levels/features</li>
<li>aliases - alternative representations of the variant</li>
</ul>
<p><strong>Example:</strong></p>
<pre><code>    &lt;variant diploid_count='1'&gt;
        &lt;gene source='HGNC' accession='AIRE'/&gt;
        &lt;ref_seq source='IDRefSeq' accession='D0003' /&gt;
        &lt;name scheme='HGVS'&gt;g.4707AT&lt;/name&gt;
        &lt;exon&gt;e1&lt;/exon&gt;
        &lt;consequence term='A point mutation in the exon 1 altering initiation codon in HSR domain '/&gt;
        &lt;seq_changes&gt;
            &lt;variant type='RNA'&gt;
                &lt;ref_seq source='IDRefSeq' accession='C0003' /&gt;
                &lt;name scheme='HGVS'&gt;r.1au&lt;/name&gt;
                &lt;consequence term='predicted to change start codon to position c.268, resulting in a frameshift of translation'/&gt;
                &lt;seq_changes&gt;
                    &lt;variant type='AA'&gt;
                        &lt;ref_seq source='uniprot' accession='O43918' /&gt;
                        &lt;name scheme='HGVS'&gt;p.Met1Leu&lt;/name&gt;
                    &lt;/variant&gt;
                &lt;/seq_changes&gt;
            &lt;/variant&gt;
        &lt;/seq_changes&gt;
    &lt;/variant&gt;
</code></pre>
  </div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>id</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>type</td><td>string</td><td>optional</td><td>DNA | cDNA | RNA | AA</td></tr>
         <tr><td>genotypic</td><td>string</td><td>optional</td><td>true</td></tr>
         <tr><td>subcellular_part</td><td>string</td><td>optional</td><td>nucleus | mitochondrial | chloroplast | other</td></tr>
         <tr><td>copy_count</td><td>float</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>gene</td><td><a href='#VmlGene'>VmlGene</a></td><td>zero or many</td><td></td></tr>
         <tr><td>ref_seq</td><td><a href='#VmlRefSeq'>VmlRefSeq</a></td><td>optional</td><td></td></tr>
         <tr><td>name</td><td><a href='#VmlName'>VmlName</a></td><td>optional</td><td></td></tr>
         <tr><td>haplotype</td><td><a href='#VmlHaplotype'>VmlHaplotype</a></td><td>zero or many</td><td></td></tr>
         <tr><td>panel</td><td><a href='#VmlPanel'>VmlPanel</a></td><td>zero or many</td><td></td></tr>
         <tr><td>seq_region</td><td><a href='#VmlSeqRegion'>VmlSeqRegion</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant_type</td><td><a href='#VmlVariantType'>VmlVariantType</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant_class</td><td><a href='#VmlVariantClass'>VmlVariantClass</a></td><td>zero or many</td><td></td></tr>
         <tr><td>original_id</td><td><a href='#VmlOriginalId'>VmlOriginalId</a></td><td>optional</td><td></td></tr>
         <tr><td>exon</td><td><a href='#VmlExon'>VmlExon</a></td><td>zero or many</td><td></td></tr>
         <tr><td>sequence</td><td><a href='#VmlSequence'>VmlSequence</a></td><td>optional</td><td></td></tr>
         <tr><td>genotype</td><td><a href='#VmlGenotype'>VmlGenotype</a></td><td>optional</td><td></td></tr>
         <tr><td>consequence</td><td><a href='#VmlConsequence'>VmlConsequence</a></td><td>zero or many</td><td></td></tr>
         <tr><td>pathogenicity</td><td><a href='#VmlPathogenicity'>VmlPathogenicity</a></td><td>zero or many</td><td></td></tr>
         <tr><td>sample</td><td><a href='#VmlSample'>VmlSample</a></td><td>optional</td><td></td></tr>
         <tr><td>tissue</td><td><a href='#VmlTissue'>VmlTissue</a></td><td>optional</td><td></td></tr>
         <tr><td>variant_detection</td><td><a href='#VmlVariantDetection'>VmlVariantDetection</a></td><td>optional</td><td></td></tr>
         <tr><td>restriction_site</td><td><a href='#VmlRestrictionSite'>VmlRestrictionSite</a></td><td>optional</td><td></td></tr>
         <tr><td>tissue_distribution</td><td><a href='#VmlTissueDistribution'>VmlTissueDistribution</a></td><td>optional</td><td></td></tr>
         <tr><td>genetic_origin</td><td><a href='#VmlGenetic_origin'>VmlGenetic_origin</a></td><td>zero or many</td><td></td></tr>
         <tr><td>frequency</td><td><a href='#VmlFrequency'>VmlFrequency</a></td><td>zero or many</td><td></td></tr>
         <tr><td>seq_changes</td><td><a href='#VmlConsequentVariants'>VmlConsequentVariants</a></td><td>optional</td><td></td></tr>
         <tr><td>aliases</td><td><a href='#VmlRelatedVariants'>VmlRelatedVariants</a></td><td>optional</td><td></td></tr>
         <tr><td>source</td><td><a href='#VmlSource'>VmlSource</a></td><td>optional</td><td></td></tr>
         <tr><td>location</td><td><a href='#VmlLocation'>VmlLocation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>sharing_policy</td><td><a href='#VmlSharingPolicy'>VmlSharingPolicy</a></td><td>optional</td><td></td></tr>
         <tr><td>creation_date</td><td><a href='#VmlCreationDate'>VmlCreationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>modification_date</td><td><a href='#VmlModificationDate'>VmlModificationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>variant_class (<a name='VmlVariantClass'><em>VmlVariantClass</em></a>)</h2>
     </p><div class='doc'></div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>variant_detection (<a name='VmlVariantDetection'><em>VmlVariantDetection</em></a>)</h2>
     </p><div class='doc'><p>Techniques used to detect the variant.</p>
<p>Variant detection method. Element has attributes:</p>
<ul>
<li>sequence template, which can be either DNA, RNA, cDNA or AA</li>
<li>detection technique</li>
</ul>
<p>Optional element protocol_id (of type DbXRef) has reference to actual detection protocol.</p>
<p>Controlled vocabulary terms for detection techniques:</p>
<ul>
<li><strong>Not Known:</strong> The information has not been recorded or provided</li>
<li><strong>Not Specified:</strong> Test information cannot be determined</li>
<li>**ARMS</li>
<li><strong>BESS:</strong> Base Excision Sequence Scanning</li>
<li><strong>CF20:</strong> CF Common Mutation Test</li>
<li><strong>CF29:</strong> Analysis of 29 mutations using the Elucigene CF29 kit</li>
<li><strong>CMC:</strong> Chemical Mismatch Cleavage</li>
<li><strong>CSCE:</strong> Conformation sensitive capillary electrophoresis</li>
<li><strong>DGGE:</strong> Denaturing-Gradient Gel-Electrophoresis</li>
<li><strong>DHPLC:</strong> Denaturing High-Performance Liquid Chromatography</li>
<li><strong>DOVAM:</strong> Detection Of Virtually All Mutations (SSCA variant)</li>
<li><strong>DSCA:</strong> Double-Strand DNA Conformation Analysis</li>
<li><strong>HD:</strong> HeteroDuplex analysis</li>
<li><strong>IHC:</strong> Immuno-Histo-Chemistry</li>
<li>**Loss of heterozygosity analysis</li>
<li><strong>mPCR:</strong> multiplex PCR</li>
<li>**Meta-PCR</li>
<li><strong>MS-PCR:</strong> Mutagenically separated PCR</li>
<li><strong>MAPH:</strong> Multiplex Amplifiable Probe Hybridisation</li>
<li><strong>MLPA:</strong> Multiplex Ligation-dependent Probe Amplification</li>
<li><strong>NGS:</strong> Next Generation Sequencing</li>
<li><strong>PAGE:</strong> Poly-Acrylamide Gel-Electrophoresis</li>
<li><strong>PCR:</strong> Polymerase Chain Reaction</li>
<li><strong>PTT:</strong> Protein Truncation Test</li>
<li><strong>RT-PCR:</strong> Reverse Transcription and PCR</li>
<li><strong>RNA:</strong> RNA work performed</li>
<li><strong>SEQ:</strong> SEQuencing</li>
<li><strong>SNPlex:</strong> The SNPlex™ Genotyping System from ABI</li>
<li><strong>Southern:</strong> Southern Blotting</li>
<li><strong>SSCA:</strong> Single-Strand DNA Conformation Analysis (SSCP)</li>
<li>**SSCP/Heteroduplex</li>
<li><strong>Western:</strong> Western Blotting</li>
</ul>
<p><em>Note: implementation of fixed vocabulary for this element is still pending.</em> See: <a href="http://www.gen2phen.org/wiki/lsdb-controlled-vocabulary-terms%22">lsdb-controlled-vocabulary-terms</a></p>
<p><strong>Example:</strong></p>
<pre><code> &lt;variant_detection template=&quot;DNA&quot; technique=&quot;DHPLC&quot;/&gt;
</code></pre>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>template</td><td>string</td><td>mandatory</td><td>DNA | RNA | cDNA | AA</td></tr>
         <tr><td>technique</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>variant (<a name='VmlVariantEvent'><em>VmlVariantEvent</em></a>)</h2>
     </p><div class='doc'><p>A variant event is a single genomic variation event, co-occurring with the given variant.</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>id</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>name</td><td><a href='#VmlName'>VmlName</a></td><td>optional</td><td></td></tr>
         <tr><td>seq_region</td><td><a href='#VmlSeqRegion'>VmlSeqRegion</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant_type</td><td><a href='#VmlVariantType'>VmlVariantType</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant_class</td><td><a href='#VmlVariantClass'>VmlVariantClass</a></td><td>zero or many</td><td></td></tr>
         <tr><td>original_id</td><td><a href='#VmlOriginalId'>VmlOriginalId</a></td><td>optional</td><td></td></tr>
         <tr><td>exon</td><td><a href='#VmlExon'>VmlExon</a></td><td>zero or many</td><td></td></tr>
         <tr><td>sequence</td><td><a href='#VmlSequence'>VmlSequence</a></td><td>optional</td><td></td></tr>
         <tr><td>consequence</td><td><a href='#VmlConsequence'>VmlConsequence</a></td><td>zero or many</td><td></td></tr>
         <tr><td>pathogenicity</td><td><a href='#VmlPathogenicity'>VmlPathogenicity</a></td><td>zero or many</td><td></td></tr>
         <tr><td>variant_detection</td><td><a href='#VmlVariantDetection'>VmlVariantDetection</a></td><td>optional</td><td></td></tr>
         <tr><td>restriction_site</td><td><a href='#VmlRestrictionSite'>VmlRestrictionSite</a></td><td>optional</td><td></td></tr>
         <tr><td>genetic_origin</td><td><a href='#VmlGenetic_origin'>VmlGenetic_origin</a></td><td>zero or many</td><td></td></tr>
         <tr><td>frequency</td><td><a href='#VmlFrequency'>VmlFrequency</a></td><td>zero or many</td><td></td></tr>
         <tr><td>seq_changes</td><td><a href='#VmlConsequentVariants'>VmlConsequentVariants</a></td><td>optional</td><td></td></tr>
         <tr><td>aliases</td><td><a href='#VmlRelatedVariants'>VmlRelatedVariants</a></td><td>optional</td><td></td></tr>
         <tr><td>location</td><td><a href='#VmlLocation'>VmlLocation</a></td><td>zero or many</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>variant_group (<a name='VmlVariantGroup'><em>VmlVariantGroup</em></a>)</h2>
     </p><div class='doc'><p>Variant group (EXPERIMENTAL)</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>id</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>orientation</td><td>string</td><td>optional</td><td>cis | trans | unknown</td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>group_type</td><td><a href='#VmlGroupType'>VmlGroupType</a></td><td>optional</td><td></td></tr>
         <tr><td>variant</td><td><a href='#VmlVariant'>VmlVariant</a></td><td>one or many</td><td></td></tr>
         <tr><td>frequency</td><td><a href='#VmlFrequency'>VmlFrequency</a></td><td>zero or many</td><td></td></tr>
         <tr><td>pathogenicity</td><td><a href='#VmlPathogenicity'>VmlPathogenicity</a></td><td>zero or many</td><td></td></tr>
         <tr><td>value</td><td><a href='#VmlValue'>VmlValue</a></td><td>zero or many</td><td></td></tr>
         <tr><td>evidence_code</td><td><a href='#VmlEvidenceCode'>VmlEvidenceCode</a></td><td>zero or many</td><td></td></tr>
         <tr><td>protocol_id</td><td><a href='#VmlProtocolId'>VmlProtocolId</a></td><td>zero or many</td><td></td></tr>
         <tr><td>observation_date</td><td><a href='#VmlObservationDate'>VmlObservationDate</a></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
     <h2 class='header'>variant_type (<a name='VmlVariantType'><em>VmlVariantType</em></a>)</h2>
     </p><div class='doc'><p>Type of variation. SNP, microsatellite... Also sequence alteration terms from SO http://www.sequenceontology.org/browser/current_release/term/SO:0001059</p>
<p>Note: This should not be mixed with classification schemes based on effects of mutations (like missense, alter splicing...) which should go into the consequence element or schemes based on type of underlying sequence (like exon, intron), which should go into seq_region element.</p>
<p><em>Examples:</em> indel (accession = 1000032) deletion (accession = 0000159) substitution (accession = 1000002) inversion (accession = 1000036)</p>
</div>
     <div class='attributes'>
       <h3>Attributes</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>value</th></tr>
         <tr><td>source</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>accession</td><td>string</td><td>optional</td><td></td></tr>
         <tr><td>uri</td><td>anyURI</td><td>optional</td><td></td></tr>
         <tr><td>term</td><td>string</td><td>mandatory</td><td></td></tr>
        </table>
     </div>
     <div class='elements'>
       <h3>Elements</h3>
       <table border='1'>
         <tr><th>name</th><th>type</th><th>cardinality</th><th>choice group</th></tr>
         <tr><td>description</td><td><em>string</em></td><td>optional</td><td></td></tr>
         <tr><td>db_xref</td><td><a href='#VmlDbXRef'>VmlDbXRef</a></td><td>zero or many</td><td></td></tr>
         <tr><td>comment</td><td><a href='#VmlComment'>VmlComment</a></td><td>zero or many</td><td></td></tr>
         <tr><td></td><td><em>choice</em></td><td>zero or many</td><td><em>Foreign XML nodes</em></td></tr>
        </table>
     </div>
<span style="font-size: .75em; float: right;">[ <a href="#top">top</a> ]</span><br/><hr/>
    </body>
</html>)
